[BUG] /usr/bin/ld: cannot find -lz

**Describe the bug**
When compiling on a fairly fresh Debian Sid instance, I got the following error upon installation: 

```
/usr/bin/ld: cannot find -lz
```

Installing `zlib1g-dev` fixes it. I'd like to add something to the README's installation dependencies to this effect, but wanted to open this issue for feedback first.

--- a/README.md
+++ b/README.md
@@ -53,13 +53,13 @@ First, install the dependencies.
 With apt:
 
 ```bash
-# apt-get install protobuf-compiler libssl-dev
+# apt-get install protobuf-compiler libssl-dev zlib1g-dev
 ```
 
 With yum
 
 ```bash
-# yum install protobuf-compiler libssl-dev
+# yum install protobuf-compiler libssl-dev zlib1g-dev
 ```
 
 ### Install `rust-ipfs` itself

